core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -893,6 +893,7 @@ class TaskSetManagerSuite extends SparkFunSuite with LocalSparkContext with Logg
     val taskSet = FakeTask.createTaskSet(4)
     // Set the speculation multiplier to be 0 so speculative tasks are launched immediately
     sc.conf.set("spark.speculation.multiplier", "0.0")
+    sc.conf.set("spark.speculation", "true")
```

Comment:
This should be set. Because the duration is inserted to `MedianHeap` only when `spark.speculation`(e.g. If I remove this, `MedianHeap` will be empty when call `checkSpeculatableTasks`).